client/apps/edit2/components/content/section_controls/test/layout.test.js

```diff
@@ -0,0 +1,153 @@
+import configureStore from 'redux-mock-store'
+import React from 'react'
+import { clone } from 'lodash'
+import { mount } from 'enzyme'
+import { Provider } from 'react-redux'
+import { LayoutControls } from '../layout.jsx'
+import { Fixtures } from '@artsy/reaction-force/dist/Components/Publishing'
+const { StandardArticle, FeatureArticle } = Fixtures
+
+describe('Section LayoutControls', () => {
+  let props
+
+  const getWrapper = (props) => {
+    const mockStore = configureStore([])
+    const store = mockStore({
+      app: {
+        channel: props.channel
+      },
+      edit: {
+        article: props.article,
+        section: props.section,
+        sectionIndex: props.sectionIndex
+      }
+    })
+
+    return mount(
+      <Provider store={store}>
+        <section>
+          <LayoutControls {...props} />
+        </section>
+      </Provider>
+    )
+  }
```

Comment:
All that said, I like this pattern and it seems to be as straightforward as can be; anyone arriving at this code would be able to immediately understand and tweak, without having to navigate through intermediary files / libs / etc  👍